### ISSUE TITLE
Show commit messages in issue 

### ISSUE THREAD

**Issue by Disane87** [waffleio/waffle.io#2771]
Currently it's not possible to show commit messages with a reference to an issue.
It would be great if my contributers will see them too, so they don't have to look into the GitHub-Issues.

**Comment by n8behavior** [comment]
Agree, my management looks at the cards to track activity.  Most of my developers commit to issues several times per day.  However, management does not see the developer activity and the card actually says things like "...comment 9 days ago" when actually there was work done just a few hours ago.  So, of course management now thinks we aren't doing anything.  My Waffle board, which is supposed to help transparency, is now misrepresenting the state of things and even hurts the project.

**Comment by jmgate** [comment]
+1

**Comment by bsmith42** [comment]
I think at the very least that the last commit shown in the card would be awesome.  It's always hard to weigh adding more data to the card but if you don't get enough info in the card then you end up going to github to get it.  That defeats the purpose.  Obvious, some deeper integration would be great but some minimums would be a great start.

**Comment by kMikaZu** [comment]
+1

**Comment by Disane87** [comment]
Wow, since two years no traction. Meh.
